release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -543,9 +543,11 @@ The assisted installer and installer-provisioned installation for bare metal clu
 If you need the virtual IP (VIP) addresses to run on the control plane nodes in a bare metal installation, you must configure the `apiVIP` and `ingressVIP` VIP addresses to run exclusively on the control plane nodes. By default, {product-title} allows any node in the worker machine configuration pool to host the `apiVIP` and `ingressVIP` VIP addresses. Because many bare metal environments deploy worker nodes in separate subnets from the control plane nodes, configuring the `apiVIP` and `ingressVIP` virtual IP addresses to run exclusively on the control plane nodes prevents issues from arising due to deploying worker nodes in separate subnets. For additional details, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configure-network-components-to-run-on-the-control-plane_ipi-install-configuration-files[Configure network components to run on the control plane].
 
 [id="ocp-4-8-networking-external-load-balancer"]
-==== Configuring an external load balancer for apiVIP and ingressVIP traffic
+==== Configuring external load balancers for apiVIP and ingressVIP traffic
 
-In {product-title} 4.8, you can configure an external load balancer to handle `apiVIP` and `ingressVIP` traffic to the control plane of installer-provisioned clusters. External load balancing services and the control plane nodes must run on the same L2 network, and on the same VLAN when using VLANs to route traffic between the load balancing services and the control plane nodes.
+In {product-title} 4.8, you can configure an external load balancer to handle `apiVIP` and `ingressVIP` traffic for {rh-openstack-first} and baremetal installer-provisioned clusters. External load balancing services and the control plane nodes must run on the same L2 network, and on the same VLAN when using VLANs to route traffic between the load balancing services and the control plane nodes.
+
+Configuring load balancers to handle `apiVIP` and `ingressVIP` traffic is not supported for VMware installer-provisioned clusters.
```

Comment:
Based on my testing https://bugzilla.redhat.com/show_bug.cgi?id=1986964#c2, external load balancers are also supported on vsphere IPI cluster, the configuration should be performed by user manually.

Reply:
@amelie1979  Based on Jinyun Ma's testing, do you reckon we need to add a comment on the RN to specify that VMware users can configure this manually using the barmetal installer procedure here? https://docs.openshift.com/container-platform/4.8/installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.html#nw-osp-configuring-external-load-balancer_ipi-install-post-installation-configuration

Reply:
This is not officially supported, since there is no way to remove the config from the operator for the builtin VMware LB and the Internal DNS will keep refer to configured api-int original IP (at installed time). Doable vs Supportable are not the same.

Reply:
https://issues.redhat.com/browse/RFE-2054

Reply:
Thank you @amelie1979 , this makes all the sense. @jinyunma please let us know if there's anything else to address, and/or give the PR your blessing to go ahead :) 

Thank you both!

Reply:
thanks @amelie1979 for clarification, then the update is look good to me @cherraz.